fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -132,10 +132,24 @@ def self.print_system_environment
       require "openssl"
 
       env_output = "### Stack\n\n"
-      product, version, build = `sw_vers`.strip.split("\n").map { |line| line.split(':').last.strip }
+      product, version, build = "Unkown"
+      os_version = "UNKOWN"
+
+      if FastlaneCore::Helper.mac?
+        product, version, build = `sw_vers`.strip.split("\n").map { |line| line.split(':').last.strip }
+        os_version = `sw_vers -productVersion`.strip
```

Comment:
Why not use the version from the line above? Those strings should be the same, I think.